CONTRIBUTING.md

```diff
@@ -273,6 +273,9 @@ Change notes should only be added as new bullet points listed under `Unreleased`
 Change notes should contain only the **user-facing** plugin alterations like new features, public-requested bug fixes, etc. <br/>
 In spite of the above, change notes can sometimes contain a general description of the work done in the new version
 if the alternative is leaving them empty, which should be avoided.
+The change notes should be in present tense (e.g. use `Add...` instead of `Added...`);
```

Comment:
Okay so we're changing the convention from now on right?

Reply:
To more git-commit-like

Reply:
Looking at the former change notes I cannot say there was any convention. 
We can go for past tense - probably it is more reasonable as in contrast to commit messages, the change notes are supposed to be read by humans (users in particular, and I do not consider developers to be humans).

Reply:
@PawelLipski wdyt?

Reply:
> and I do not consider developers to be humans
:uuu:

> We can go for past tense 
Yeah, let's keep as it is

Reply:
> Yeah, let's keep as it is

this order is not clear to me

Reply:
Sorry ;p keep past tense